### PR TITLE
fix few css bugs; minor client refactoring

### PR DESCRIPTION
**Pull request by lmssiehdev**

### CODE REVIEW

File: client/src/api.ts
Comment (leia-uwu) [inline client/src/api.ts:1]: i would like to keep api.ts because it works like a config for the api URL

File: client/l10n/en.json
Comment (leia-uwu) [inline client/l10n/en.json:1]: i don't think some translations keys from here should have been deleted
like `index-google` (when we add accounts i want google and discord login support, don't care about others tho),

**Comment by leia-uwu** [comment]
also idk about binding equip other gun to something by default
i'm a fan of that but don't want to keep adding exceptions to 1:1 remake xd

**Comment by lmssiehdev** [comment]
Both the "Equip Other Gun" and space weren’t set to anything, so it’s not really an issue, and most players set it themselves. I think it’s still worth making it the default.

I won’t revert it, but feel free to do so.

**Pull request merged**